unfoldingWord/gateway-translate
# bumped oce-editor-tools version

- bug fix for undo/redo

## Test Instructions

- open a book in gateway-translate
- make an edit to the main text then undo
- make another change and blur
- make another change and blur again.

No longer causing duplication of every word
